rpcserver_test.go

```diff
@@ -1,14 +1,79 @@
 package lnd
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/fn"
+	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
 )
 
 func TestGetAllPermissions(t *testing.T) {
 	perms := GetAllPermissions()
 
-	// Currently there are there are 16 entity:action pairs in use.
+	// Currently there are 16 entity:action pairs in use.
 	assert.Equal(t, len(perms), 16)
 }
+
+// mockDataParser is a mock implementation of the AuxDataParser interface.
+type mockDataParser struct {
```

Comment:
use `mock.Mock` instead?

Reply:
Same answer as above: Happy to help do that outside the context of this PR.